packages/react-table/src/docs/examples/Data.jsx

```diff
@@ -0,0 +1,917 @@
+/* eslint-disable no-console */
+import * as React from 'react';
+import _ from 'lodash';
+
+export const columns = [
+  { title: 'Servers' },
+  { title: 'Threads' },
+  { title: 'Applications' },
+  { title: 'Workspaces' },
+  { title: 'Status' },
+  { title: 'Location' },
+  { title: 'Last Modified' },
+  { title: 'URL' }
+];
+
+export const rows = [
```

Comment:
I had expected the table data to be formatted more the way we'd expect data to be returned from an api call. so that each object would have the same declared type like the examples do and be formatted more like:
```
datum: {
      server: 'US-Node 1',
      threads: _.random(1, 20),
      applications: _.random(1, 50),
      workspaces: _.random(1, 30),
      status: _.sample(['Stopped', 'Running', 'Down', 'Needs Maintenance']),
      location: _.sample(['Raleigh', 'Boston', 'Atlanta', 'San Francisco']),
      last modified: '2 hours ago',
      url: 'http://www.redhat.com/en/office-locations/US-node1'
    }
```